index.html

```diff
@@ -189,26 +205,34 @@ <h2 class="skill-title color1">Skills</h2>
             </div>
         </section>
         <section class="contact">
+            <div class="contact-heading-container">
                 <h2 class="primary color1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            </div>
```

Comment:
- [ ] kindly increase the space above the form to make your design an exact duplicate of the given design 
<img width="1407" alt="Screen Shot 2023-06-17 at 23 14 20" src="https://github.com/BHObserver/portfolio/assets/60146030/831d396c-0e83-4d02-8804-d9269d9ded12">